recipes/cdflib/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "cdflib" %}
+{% set version = "0.3.17" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9b027b0e5b7d84816aafa13edc5df3767c560881e8a93348f92cdfba29d7897a
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
```

Comment:
I don't know what part of the installation requires `twine`, but I would suggest you to add `twine >=1.11` in the `host` section.